packages/fast-components-react-base/src/tabs/examples.data.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from "react";
+import { ISnapshotTestSuite } from "@microsoft/fast-jest-snapshots-react";
+import { IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
+import Tabs, {
+    ITabsHandledProps,
+    ITabsManagedClasses,
+    ITabsUnhandledProps,
+    Tab,
+    TabItem,
+    TabPanel
+} from "./tabs";
+import schema from "./tabs.schema.json";
+import Documentation from "./.tmp/documentation";
+import { Orientation } from "./tabs.props";
+
+function getTabItem(index: number): JSX.Element {
+    return (
+        <TabItem slot="tab-item" id={`tab0${index}`}>
+            <Tab slot="tab">
```

Comment:
It seems a little odd to me that we have a `Tab` component that requires a `slot={"tab"}`. Do we really even need `Tab` and `TabPanel` components? I don't really follow the purpose they serve.

Reply:
They are basically buckets in which there is an opportunity for children to be placed. So for example if I wanted to put an SVG in the `Tab` I can do that as well as any content I want in the `TabPanel`. The prop of slot is to follow our convention that we are going to be syncing this up with an API for web components in the future so in order to find an appropriate slot I need the prop. Also there's no way of knowing once the components are minified what they are, in fact you could use any element or a custom component as long as you have the slot prop with the correct string.

Reply:
Additionally `TabItem` must contain a `Tab` and a `TabPanel` in order to be a complete item, that's really what the purpose is. If it does not then we don't count it as a viable `TabItem`.